var/spack/repos/builtin/packages/stream/package.py

```diff
@@ -1,42 +0,0 @@
-# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
-# Spack Project Developers. See the top-level COPYRIGHT file for details.
-#
-# SPDX-License-Identifier: (Apache-2.0 OR MIT)
-
-from spack import *
-
-
-class Stream(MakefilePackage):
```

Comment:
Was this package removed by accident or on purpose?

Reply:
Stream test is still on, so I deleted stream recipe purposefully

Reply:
I'm not sure I understand, why do we want to remove this package for all users?

Reply:
My bad, I thought local updation of stream was removed.

I have now readded the stream again.